CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 # Revision history for plutarch
 
+# 1.1.0
+
+- Bunch of new functions.
```

Comment:
Can we list those functions here? Or at least the modules they were added in.

Reply:
Sure, if somebody makes a PR we can merge it.

Reply:
I was working on a detailed changelog - but it probably will have to wait